ngraph/frontend/ir/src/model.cpp

```diff
@@ -425,18 +433,30 @@ ngraph::op::v5::Loop::SpecialBodyPorts XmlDeserializer::parsePurposeAttribute(co
 void XmlDeserializer::on_adapter(const std::string& name, ngraph::ValueAccessor<void>& adapter) {
     static const std::unordered_set<std::string> skip_names = {"input_descriptions",
                                                                "output_descriptions",
-                                                               "special_body_ports"};
+                                                               "special_body_ports",
+                                                               "then_inputs",
+                                                               "else_inputs",
+                                                               "then_outputs",
+                                                               "else_outputs"};
     std::string val;
 
     // for TensorIterator look for 'port_map' as 'data' does not exist
-    if (m_node.child("port_map")) {
-        if (auto a = ngraph::as_type<
-                ngraph::AttributeAdapter<std::vector<std::shared_ptr<ngraph::op::util::SubGraphOp::InputDescription>>>>(
-                &adapter)) {
-            a->set(parseInputDescription(m_node));
+    if (m_node.child("port_map") || m_node.child("then_port_map") || m_node.child("else_port_map")) {
```

Comment:
This is the third place where we have this names. Could you consolidate all this logic and string values under some common structure to avoid duplicates and possible mistakes in naming.

Reply:
@GlebKazantaev model.cpp and serialize.cpp are in different libraries, I can only implement public (OPENVINO_API) helper func in some shared utils file.